services/static-webserver/client/source/class/osparc/component/permissions/Service.js

```diff
@@ -130,17 +130,17 @@ qx.Class.define("osparc.component.permissions.Service", {
       osparc.data.Resources.fetch("services", "patch", params)
         .then(serviceData => {
           this.fireDataEvent("updateService", serviceData);
-          osparc.component.message.FlashMessenger.getInstance().logAs(this.tr("Collaborator successfully removed"));
+          osparc.component.message.FlashMessenger.getInstance().logAs(this.tr("Member successfully removed"));
           this.__reloadOrganizationsAndMembers();
           this.__reloadCollaboratorsList();
         })
         .catch(err => {
-          osparc.component.message.FlashMessenger.getInstance().logAs(this.tr("Something went wrong removing Collaborator"), "ERROR");
+          osparc.component.message.FlashMessenger.getInstance().logAs(this.tr("Something went wrong removing Member"), "ERROR");
```

Comment:
why not displaying the error message coming response? The openapi should show specs on how these errors are formatted

Reply:
The only way the frontend has for updating the access rights is by 'put'ting the entire study/service. I doubt that the error message will give a more clear text

Reply:
Ahah I see your point. True. Actually you mentioned to me that limitation